### PR TITLE
Added Array

### PR DESCRIPTION
**Pull request by PrenaA**
Added Array

### CODE REVIEW

File: src/test/resources/Ds-algoFeatures/Array.feature [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+  @tag5 Array
+  Background: The user is logged in to DS Algo portal
+  Scenario Outline: The user is able run code in tryEditor of Arrays in Python page
+    Given The user is in a page having an tryEditor with a Run button to test
+    When "The user enter valid python code in tryEditor from sheet "print "Welcome to Ds-Algo"" and "<1>"
+		And The user clicks on run button
+		Then "The user should be presented with Run result"
+		
+		
+	@tag6 Array
```
Comment (vasavibodepu) [inline src/test/resources/Ds-algoFeatures/Array.feature:61]: why tag6 doesn't show in red indentation ?.
Have you tested this feature in local env ?..

File: src/test/resources/Ds-algoFeatures/Array.feature [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+    Given The user is in a page having an tryEditor with a Run button to test
+    When The user enter valid python code in tryEditor with valid credential 
+		And The user clicks on run button
+ 		Then The user should be presented with Run result
+   
+   Examples: 
+      | Sheetname  | RowNumber |
+      | pythonCode |         0 |
+    
+  @tag10 Array
```
Comment (vasavibodepu) [inline src/test/resources/Ds-algoFeatures/Array.feature:102]: same comment as tag6

**Review by vasavibodepu — COMMENTED** [review]
please close the commented issues

File: src/test/resources/Ds-algoFeatures/Array.feature (hunk 1 above)
Comment (PrenaA) [reply src/test/resources/Ds-algoFeatures/Array.feature:61]: i did some changes in Array feature file

File: src/test/resources/Ds-algoFeatures/Array.feature (hunk 2 above)
Comment (PrenaA) [reply src/test/resources/Ds-algoFeatures/Array.feature:102]: I am done

**Review by vasavibodepu — APPROVED** [review]

**Pull request merged**